src/include/distributed/commands/multi_copy.h

```diff
@@ -169,7 +177,8 @@ extern CitusCopyDestReceiver * CreateCitusCopyDestReceiver(Oid relationId,
 														   int partitionColumnIndex,
 														   EState *executorState,
 														   char *intermediateResultPrefix,
-														   bool isPublishable);
+														   bool isPublishable,
+														   bool skipCopyToLocalPlacements);
```

Comment:
to keep the API somewhat simple, maybe we could skip having it as a parameter?

callers can always set it in the struct after creating it

Reply:
https://github.com/citusdata/citus/pull/7131#discussion_r1302935031